app/assets/stylesheets/styles/_presets.css.scss

```diff
@@ -143,6 +148,25 @@ textarea {
   float: none;
 }
 
+// Must reside in the top level of stylesheets
+// Note: sticky-footer is setting the height of the first selector to auto !important. To counteract that we are setting it manually.
+@include sticky-footer($footer-height, "#foo-bar", "#layout_footer", "#footer");
+#content {
```

Comment:
Haha it's fine. Mainly me complaining. I think this warning is good, and the language of "avoid" is true, but not here.

I don't know how I feel about the comments thing. I see nothing wrong with `/* */` comments?